cluster-provision/ocp/4.3/publish.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+docker tag kubevirtci/ocp-4.3-provision:latest quay.io/openshift-cnv/kubevirtci-ocp-4.3:latest
+docker push quay.io/openshift-cnv/kubevirtci-ocp-4.3:latest
```

Comment:
I understand this repo is private? If so, how would we login here?

Reply:
docker login, same we do with okd.